docs/source/contribution.rst

```diff
@@ -275,19 +275,19 @@ To build the documentation, you need to install `Sphinx <http://www.sphinx-doc.o
 
   $ pip install sphinx sphinx_rtd_theme
 
-Then you can build the documentation in HTML format locally::
+
+.. note::
+
+   Docstrings (documentation comments in the source code) are collected from the installed Chainer module. If you have edited docstrings in checked-out source files and want to see those changes reflected in the generated html, Chainer must be installed in develop mode to see those changes reflected in the generated documentation. To do this use `pip install -e .`from the the top of the Chainer directory.
+
+Move to the docs directory, and make the html::
```

Comment:
The word "html" appears here at the first time, so "the html" sounds a bit too sudden. 
I think the original sentence was okay.